src/nextjournal/clerk/view.clj

```diff
@@ -3,31 +3,44 @@
             [nextjournal.clerk.viewer :as v]
             [hiccup.page :as hiccup]
             [clojure.string :as str]
-            [clojure.java.io :as io]))
+            [clojure.java.io :as io])
+  (:import (java.net URI)))
 
 (defn doc->viewer
   ([doc] (doc->viewer {} doc))
   ([opts {:as doc :keys [ns file]}]
    (binding [*ns* ns]
      (-> (merge doc opts) v/notebook v/present))))
 
-
 #_(doc->viewer (nextjournal.clerk/eval-file "notebooks/hello.clj"))
 #_(nextjournal.clerk/show! "notebooks/test.clj")
 #_(nextjournal.clerk/show! "notebooks/visibility.clj")
 
-(defn include-viewer-css []
+(defn relative? [url]
+  (and (not (.isAbsolute (URI. url)))
+       (not (str/starts-with? url "/"))))
```

Comment:
Why are both of these checks needed?

Reply:
Because `(= false (.isAbsolute (URI. "/viewer.css")))`